android/gradle.properties

```diff
@@ -38,11 +33,11 @@ reactNativeArchitectures=armeabi-v7a,arm64-v8a,x86,x86_64
 # your application. You should enable this flag either if you want
 # to write custom TurboModules/Fabric components OR use libraries that
 # are providing them.
-newArchEnabled=false
+newArchEnabled=true
 
 # Use this property to enable or disable the Hermes JS engine.
 # If set to false, you will be using JSC instead.
-hermesEnabled=true
+hermesEnabled=false
```

Comment:
SES is currently disabled in android (we need to enable it if it's working, but not in this PR), so we can enable hermes

Performance without hermes in android is terrible

Reply:
Thanks for the heads up! Re-enabled hermes without any issue in this new version.